### PR TITLE
update coding style in c++

### PR DESCRIPTION
**Pull request by iattempt**
**Fixes issue:**

**Changes:**
1. add const and pointer/reference style guide.
2. use easier simple to demonstrate.

### CODE REVIEW

File: guides/coding_style/c++/README.md [hunk 1]
```diff
@@ -82,14 +82,61 @@ bool
 operator==(container<T,Allocator> x, container<T,Allocator> y);
 
 // n > 100 columns
-void cloneTwo(const container<T,Allocator>& x,
-              const container<T,Allocator>& y,
-              const container<T,Allocator>& z);
+void cloneTwo(container<Type, Allocator> xxx,
```
Comment (arnavb) [inline guides/coding_style/c++/README.md]: Better names for all three would be foo, bar, and baz.

File: guides/coding_style/c++/README.md [hunk 2]
```diff
@@ -82,14 +82,61 @@ bool
...
 void
-cloneTwo(const container<T,Allocator>& x,
-         const container<T,Allocator>& y,
-         const container<T,Allocator>& z);
+cloneTwo(container<Type, Allocator> xxx,
+         container<Type, Allocator> yyy,
+         container<Type, Allocator> zzz);
+```
+
+- The pointers `*` `**` and references `&` `&&` `*&` should be part of variable name (also if variable name is not declared) or qualifier. Add space between previous type or qualifier (unless `**`, `&&`, and `*&`).
```
Comment (arnavb) [inline guides/coding_style/c++/README.md]: _the_ variable name. Add _a_ space between previous _types_ or _qualifiers_.

File: guides/coding_style/c++/README.md [hunk 3]
```diff
@@ -82,14 +82,61 @@ bool
...
+{
+    // ...
+}
+
+operator==(int *&, int *&);
+
+operator==(int &&, int &&);
+```
+
+- The `const` qualifiers should be on the right hand of its target but be left of type.
```
Comment (arnavb) [inline guides/coding_style/c++/README.md]: Right hand _side_ if it’s target but be on the left of the type.

**Comment by arnavb** [comment]
Just some minor grammar issues.

File: guides/coding_style/c++/README.md [hunk 4]
```diff
@@ -82,14 +82,61 @@ bool
...
+{
+    // ...
+}
+
+operator==(int *&, int *&);
+
+operator==(int &&, int &&);
+```
+
+- The `const` qualifiers should be on the right hand side of its target but be on left of the type.
```
Comment (arnavb) [inline guides/coding_style/c++/README.md]: qualifier should be singular.


... but to the left of the type.

File: guides/coding_style/c++/README.md [hunk 5]
```diff
@@ -82,14 +82,61 @@ bool
...
 void
-cloneTwo(const container<T,Allocator>& x,
-         const container<T,Allocator>& y,
-         const container<T,Allocator>& z);
+cloneTwo(container<Type, Allocator> foo,
+         container<Type, Allocator> bar,
+         container<Type, Allocator> baz);
+```
+
+- The pointers `*` `**` and references `&` `&&` `*&` should be part of the variable name (also if variable name is not declared) or qualifier. Add a space between previous types or qualifiers (unless `**`, `&&`, and `*&`).
```
Comment (arnavb) [inline guides/coding_style/c++/README.md]: also if -> even if

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**